sw/device/exts/common/flash_link.ld

```diff
@@ -0,0 +1,115 @@
+/* Copyright lowRISC contributors. */
+/* Licensed under the Apache License, Version 2.0, see LICENSE for details. */
+/* SPDX-License-Identifier: Apache-2.0 */
+
+/**
+ * Linker script for an OpenTitan flash binaries.
+ *
+ * Portions of this file are Ibex-specific.
+ */
+
+OUTPUT_ARCH(riscv)
+GROUP(-lgcc)
+
+/**
+ * Indicate that there are no dynamic libraries, whatsoever.
+ */
+__DYNAMIC = 0;
+
+MEMORY {
+  ram(w)    : ORIGIN = 0x10000000, LENGTH = 0x10000
+  flash(rx) : ORIGIN = 0x20000000, LENGTH = 0x100000
+}
+
+/**
+ * The stack starts at the end of RAM and grows down.
+ */
+_stack_start = ORIGIN(ram) + LENGTH(ram);
+
+SECTIONS {
+  /**
+   * C runtime (CRT) section, containing program initialization code.
+   *
+   * We don't use ENTRY, so the start of |flash| acts as the entrypoint. Since
+   * the CRT is the first thing that needs to run, it goes at the top.
+   */
+  .crt ORIGIN(flash): {
+    KEEP(*(.crt))
+    *(.crt)
+  } > flash
+
+  /**
+   * Ibex interrupt vector. See sw/device/lib/irq_vectors.S for more information.
+   *
+   * The 256-bit alignment is required by Ibex's vectoring mechanism.
+   */
+  .vectors : ALIGN(0x100) {
+    _vectors_start = .;
+    KEEP(*(.vectors))
+    *(.vectors)
+  } > flash
+
+  /**
+   * Standard text section, containing program code.
+   */
+  .text : ALIGN(4) {
+    *(.text)
+    *(.text.unlikely)
```

Comment:
Let's include `.text.hot` in this section (It might be created by the same parts of GCC that create `.text.unlikely`).

Reply:
Are there any `.text.*` things we don't want here?

Reply:
Let's just do `.text.*`. I think our policy for linker scripts should be to "do something that looks super obvious" and write comments whenever we have to go out of our way, such as with .idata